### DISCUSSION TITLE
API V2 Episode endpoint fails

### DISCUSSION THREAD

**Discussion by AdalynBlack** [amvstrm/amvstrm#40] [Discussions]
The episode endpoint is failing for every ID I test with it, displaying this screen instead of the expected API response: 

![image](https://github.com/amvstrm/amvstrm/assets/32852821/d24db391-068d-400f-a7e1-7611b0651f02)

I had a look around, and seeing as nothing has automatically flagged it, and there is no mention of it here, I've decided to post it here to make sure that the devs are aware of the issue

**Comment by NYT92** [comment]
Hello, I have fixed the issue and deployed them already but I haven't committed the fix one yet and yes I am aware of that issue right now.

The request might be cache too so the error might be gone if you refresh or disable cache.

**Reply by AdalynBlack** [reply]
I see
I'll mess around with caching to see if I can resolve the issue that way

**Reply by AdalynBlack** [reply]
It seems that refreshing the cache, at least on my browser, didn't help. I'm using the `api.amvstr.me` address to access the API, so perhaps it needs to update there first, or maybe some redirect to the website around me has cached the data and isn't updating it. Either way, if it's just a caching issue, it should be fixed soon, so it's not too big of a deal

**Reply by NYT92** [reply]
Ah ok then, If the issue still occurs I might have to look into it by myself and also committed the code after that.
